### ISSUE TITLE
NeuralNetFastAI training fails when presets is good_quality_faster_inference_only_refit

### ISSUE THREAD

**Issue by songqiang** [awslabs/autogluon#967]
The FastAI neural network models fails when preset is good_quality_faster_inference_only_refit or high_quality_fast_inference_only_refit. It works fine when presets is medium_quality_faster_train or best_quality.

Package version:
```
d5f970e - (HEAD -> master, awslab/master) [Text Fix] Fix the competition link + Fix docs (#963) (2 hours ago) <Xingjian Shi>
```

Code to reproduce error
```
df = pd.DataFrame({
    'y': [1, 2, 1, 2, 3] * 100, 
    'x': [3, 4, 3, 4, 3] * 100,
    's': ['a', 'b', 'c', 'a', 'c'] * 100,
})

hyperparameters = {
    'FASTAI': {},
    'RF': {},
}
predictor_test = TabularPredictor(
    label=label,
    verbosity=2,
    path = 'autogluon-models/uph-test/',
).fit(
    train_data=df,
    hyperparameters=hyperparameters,
#    presets = 'high_quality_fast_inference_only_refit',
    presets = 'good_quality_faster_inference_only_refit',
#    presets = 'medium_quality_faster_train',
)
predictor_test.get_model_names()
```

The error message looks like:
```
2021-02-26 07:25:44,791: [_train_and_save] Fitting model: NeuralNetFastAI_BAG_L1 ...

Epoch 22: early stopping
█

2021-02-26 07:26:23,088: [_train_and_save] 	Warning: Exception caused NeuralNetFastAI_BAG_L1 to fail during training... Skipping this model.
2021-02-26 07:26:23,089: [_train_and_save] 		'NoneType' object has no attribute 'export'
2021-02-26 07:26:23,089: [_train_and_save] Detailed Traceback:
Traceback (most recent call last):
  File "/home/ec2-user/efs/home/qiangson/workspace/autogluon/tabular/src/autogluon/tabular/trainer/abstract_trainer.py", line 912, in _train_and_save
    model = self._train_single(X, y, model, X_val, y_val, **model_fit_kwargs)
  File "/home/ec2-user/efs/home/qiangson/workspace/autogluon/tabular/src/autogluon/tabular/trainer/abstract_trainer.py", line 882, in _train_single
    model.fit(X=X, y=y, **model_fit_kwargs)
  File "/home/ec2-user/efs/home/qiangson/workspace/autogluon/core/src/autogluon/core/models/abstract/abstract_model.py", line 309, in fit
    self._fit(**kwargs)
  File "/home/ec2-user/efs/home/qiangson/workspace/autogluon/core/src/autogluon/core/models/ensemble/stacker_ensemble_model.py", line 137, in _fit
    super()._fit(X=X, y=y, k_fold=k_fold, k_fold_start=k_fold_start, k_fold_end=k_fold_end, n_repeats=n_repeats, n_repeat_start=n_repeat_start, time_limit=time_limit, **kwargs)
  File "/home/ec2-user/efs/home/qiangson/workspace/autogluon/core/src/autogluon/core/models/ensemble/bagged_ensemble_model.py", line 225, in _fit
    self.save_child(fold_model, verbose=False)
  File "/home/ec2-user/efs/home/qiangson/workspace/autogluon/core/src/autogluon/core/models/ensemble/bagged_ensemble_model.py", line 355, in save_child
    child.save(verbose=verbose)
  File "/home/ec2-user/efs/home/qiangson/workspace/autogluon/tabular/src/autogluon/tabular/models/fastainn/tabular_nn_fastai.py", line 363, in save
    save_pkl.save_with_fn(f'{path_final}{self.model_internals_file_name}', self.model, lambda m, buffer: m.export(buffer), verbose=verbose)
  File "/home/ec2-user/efs/home/qiangson/workspace/autogluon/core/src/autogluon/core/utils/savers/save_pkl.py", line 39, in save_with_fn
    pickle_fn(object, fout)
  File "/home/ec2-user/efs/home/qiangson/workspace/autogluon/tabular/src/autogluon/tabular/models/fastainn/tabular_nn_fastai.py", line 363, in <lambda>
    save_pkl.save_with_fn(f'{path_final}{self.model_internals_file_name}', self.model, lambda m, buffer: m.export(buffer), verbose=verbose)
AttributeError: 'NoneType' object has no attribute 'export'
```

**Comment by Innixma** [comment]
Thanks for spotting this! This bug also affected the AG_TEXT_NN model. I have fixed both in this PR: #973

**Comment by Innixma** [comment]
#973 is merged and fixes this issue.
